genestack_client/scripts/genestack_user_setup.py

```diff
@@ -38,6 +38,52 @@ def ask_alias(existed):
         return alias
 
 
+def ask_use_token():
+    items = ['by token', 'by email and password']
+
+    while True:
+        for i, option in enumerate(items, start=1):
+            print '%s) %s' % (i, option)
+
+        raw_index = raw_input("Select authentication: ").strip()
+
+        try:
+            user_index = int(raw_index) - 1
+        except ValueError:
+            print 'Wrong number: "%s".' % raw_index
```

Comment:
Belongs to `stderr`.

Reply:
This is not error, it is dialog with user, we should keep in in single output.